src/packages/packages.controller.ts

```diff
@@ -0,0 +1,144 @@
+import { 
+  Controller, 
+  Get, 
+  Param, 
+  StreamableFile, 
+  NotFoundException, Res,
+  BadRequestException
+} from '@nestjs/common';
+import { EntityManager } from '@mikro-orm/postgresql';
+import { Package } from '../model/package.entity';
+import { Response } from 'express';
+import * as stream from 'stream';
+import { Download } from 'src/model/download.entity';
+import { Version } from 'src/model/version.entity';
+
+@Controller('api/v1/packages')
+export class PackagesController {
+  constructor(private readonly em: EntityManager) {}
+  @Get()
```

Comment:
Let's add optional `limit` query param and. By default 10.  Use it then pls for finding Packages to limit results